recipe/meta.yaml

```
@@ -179,54 +178,285 @@ requirements:
+        - xcb-util-renderutil                # [linux]
+        - xorg-libx11                        # [linux]
+        - xorg-libxext                       # [linux]
+        - xorg-libxdamage                    # [linux]
+        - xorg-libsm                         # [linux]
+        - xorg-libice                        # [linux]
+        # https://github.com/conda-forge/staged-recipes/pull/22084
+        - xorg-libxxf86vm                    # [linux]
+        - xorg-xf86vidmodeproto              # [linux]
+        # qtlocation requires mapbox-gl-native that uses libbost-headers
+        - libboost-headers
+        - {{ pin_subpackage('libqt5', exact=True) }}
+      run:
+        - {{ pin_subpackage('libqt5', exact=True) }}
+        - {{ pin_compatible("nss") }}        # [unix]
+        - {{ pin_compatible("nspr") }}       # [unix]
+        - libxcb                             # [linux]
+        # Adding the following xcb-util run requirements resolves issue
+        # that may exhibit as segfault due to missing libraries
+        # in some lighter weight linux installations
+        # https://github.com/conda-forge/qt-main-feedstock/issues/32
+        - xcb-util-wm                        # [linux]
+        - xcb-util-keysyms                   # [linux]
+        - xcb-util-image                     # [linux]
+        - xcb-util-renderutil                # [linux]
+        # https://github.com/conda-forge/xorg-libx11-feedstock/pull/28
+        - xorg-libx11                        # [linux]
+        # https://github.com/conda-forge/xorg-libxext-feedstock/pull/12
+        - xorg-libxext                       # [linux]
+        # no run_export
+        - xorg-libsm                         # [linux]
+        - xorg-libice                        # [linux]
+        - xorg-xf86vidmodeproto              # [linux]
+      run_constrained:
+        - qt {{ version }}
+    test:
+      requires:
+        - {{ compiler('cxx') }}
+        - libgl-devel                        # [linux]
+        - libegl-devel                       # [linux]
+        - make                               # [unix]
+        - xcb-util                           # [linux]
+        - xorg-libxdamage                    # [linux]
+        # https://github.com/conda-forge/staged-recipes/pull/22084
+        - xorg-libxxf86vm                    # [linux]
+        # llvm<16 require we manually add libcxx-devel on osx,
+        # see https://github.com/conda-forge/qt-main-feedstock/pull/288#issuecomment-2351649435
+        - libcxx-devel                       # [osx]
+      files:
+        - test/hello.pro
+        - test/main-qtwebengine.cpp
+        - test/main.cpp
+        - test/main.qml
+        - test/qml.qrc
+        - test/qrc_qml.cpp
+        - test/qtwebengine.pro
+        - test/test_qmimedatabase.pro
+        - test/test_qmimedatabase.cpp
+        - xcodebuild
+        - xcrun
+
+  # Only what's needed for run_exports downstream
+  - name: libqt5
```

This should be for the same reason why the main output is `qt-main` instead of `qt`
```suggestion
  - name: libqt5-main
```